src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -36,7 +36,7 @@ struct CPUStreamsExecutor::Impl {
                 : custom::task_scheduler_observer(arena),
                   _mask{std::move(mask)},
                   _ncpus(ncpus),
-                  _cpu_ids(cpu_ids) {}
+                  _cpu_ids(std::move(cpu_ids)) {}
```

Comment:
Will move work here? The variable is `const std::vector<int>`, usually move is not working for const objects.